lib/index.js

```diff
@@ -3,23 +3,39 @@
  */
 'use strict';
 
-const {documentLoader} = require('bedrock-jsonld-document-loader');
-const jsigs = require('jsonld-signatures');
-const {Ed25519Signature2018} = jsigs.suites;
+const {jsonLdDocumentLoader} = require('bedrock-jsonld-document-loader');
+const {Ed25519Signature2020} = require('@digitalbazaar/ed25519-signature-2020');
 const {validateOperation, runOperation} = require('webkms-switch');
 const keystores = require('./keystores.js');
 const BedrockKmsModuleManager = require('./BedrockKmsModuleManager.js');
 const MongoDBKeyDescriptionStorage =
   require('./MongoDBKeyDescriptionStorage.js');
+
+const ed25519sig2018 = require('ed25519-signature-2018-context');
+const ed25519sig2020 = require('ed25519-signature-2020-context');
+const x25519 = require('x25519-key-agreement-2020-context');
```

Comment:
I think the `x25519` var here should be more consistent with the others, maybe:

```suggestion
const ed25519sig2018 = require('ed25519-signature-2018-context');
const ed25519sig2020 = require('ed25519-signature-2020-context');
const x25519ka2020 = require('x25519-key-agreement-2020-context');
```

Reply:
Updated here 425faee